src/test/java/com/algorand/algosdk/unit/IndexerPaths.java

```diff
@@ -333,4 +333,23 @@ public void searchForApplications(String string) {
         ps.q = q;
     }
 
+    @When("we make a LookupApplicationBoxByIDandName call with applicationID {long} with encoded box name {string}")
+    public void lookUpApplicationBox(Long appID, String boxName) {
+        ps.q = indexerClient.lookupApplicationBoxByIDandName(appID).name(boxName);
+    }
+
+    @When("we make a SearchForApplicationBoxes call with applicationID {long} with max {long}")
```

Comment:
I think this step method can be removed now

Reply:
good find, just removed.